kola/tests/kubeadm/templates.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2021 Kinvolk GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package kubeadm
+
+var (
+	masterScript = `#!/bin/bash
+export CNI_VERSION={{ .CNIVersion }}
+export CRICTL_VERSION={{ .CRIctlVersion }}
+export RELEASE_VERSION={{ .ReleaseVersion }}
+export DOWNLOAD_DIR={{ .DownloadDir }}
+export RELEASE={{ .Release }}
+export PATH="${PATH}:${DOWNLOAD_DIR}"
+
+# this block redirects everything to /dev/null in order to print out only the final
```

Comment:
This makes it hard to catch any `curl` errors. In general I would say `set -euo pipefail` should be the first thing that script does to run in a stricter mode.